builtin.py

```diff
@@ -58,6 +59,34 @@ def advance(self, index: int) -> Node:
     def __str__(self):
         return self.title
 
+class Obj(Node):
```

Comment:
I would change the name of the Obj class (in python, you can pass Object as a constructor, which may lead to confusion). I would call it the Item class.

Reply:
Changed to Item